modules/clear-with-shortcut/src/main/java/com/codeborne/selenide/clearwithshortcut/ClearWithShortcut.java

```diff
@@ -75,7 +77,12 @@ protected void clearWithShortcut(Driver driver, WebElement input) {
   @Nonnull
   @CheckReturnValue
   protected Platform getPlatform(Driver driver) {
-    if (!(driver.getWebDriver() instanceof JavascriptExecutor)) {
+    WebDriver webDriver = driver.getWebDriver();
+
+    if (webDriver instanceof WrapsDriver)
```

Comment:
Maybe it will be better move this check to https://github.com/selenide/selenide/blob/master/src/main/java/com/codeborne/selenide/Driver.java#L51 and use it here like `if(!driver.supportsJavascript())`?

Reply:
I was thinking to change getWebDriver method behavior, but you are right, instanceOf check better to replace with supportsJavascript() call